pkg/system/chtimes.go

```diff
@@ -2,14 +2,23 @@ package system
 
 import (
 	"os"
+	"syscall"
 	"time"
+	"unsafe"
 )
 
 // Chtimes changes the access time and modified time of a file at the given path
 func Chtimes(name string, atime time.Time, mtime time.Time) error {
 	unixMinTime := time.Unix(0, 0)
-	// The max Unix time is 33 bits set
-	unixMaxTime := unixMinTime.Add((1<<33 - 1) * time.Second)
+	var unixMaxTime time.Time
```

Comment:
Looks like this can be moved to pkg-level and then reused in tests as well. WDYT?